trussworks/react-uswds
# fix: Added hook to combobox for when options change

# Summary

- added hook to combobox for when options change
- added test
- updated story to use this feature
- added veggies to food list and renamed file

## Related Issues or PRs

Resolves #2182

## How To Test

- Run new test
- See updated story

### Screenshots (optional)
![Jan-19-2024 10-59-17](https://github.com/trussworks/react-uswds/assets/764090/1e31e3dc-f920-45ac-a52c-897f5c292a20)
